src/Sentry/SentryStackFrame.cs

```diff
@@ -182,7 +182,7 @@ public void ConfigureAppFrame(SentryOptions options)
         {
             ConfigureAppFrame(options, Function, mustIncludeSeparator: true);
         }
-        else
+        else if (ImageAddress is null or 0 && InstructionAddress is null or 0) // Leave InApp=null on NativeAOT
```

Comment:
Just curious... why does this condition imply it's not a NativeAOT app?

Reply:
https://github.com/getsentry/sentry-dotnet/blob/feat/4.0.0/src/Sentry/Internal/DebugStackTrace.cs#L237

Reply:
I'm not all the way up to speed but a simple `0` works? The framework returns an `IntPtr`, the `InstructionAddress` is a `nint`.

Reply:
it's nullable so I have to check for that first